src/codata2022.jl

```diff
@@ -0,0 +1,115 @@
+module CODATA2022
```

Comment:
This file began as a derivative of `src/codata2018.jl` and was updated to include the recently-published values.